sqlmesh/core/snapshot/definition.py

```diff
@@ -194,8 +194,12 @@ def snapshot_id(self) -> SnapshotId:
 
     @property
     def qualified_view_name(self) -> QualifiedViewName:
-        catalog, schema, table = parse_model_name(self.name)
-        return QualifiedViewName(catalog=catalog, schema_name=schema, table=table)
+        view_name = exp.to_table(self.name)
+        return QualifiedViewName(
+            catalog=view_name.catalog or None,
```

Comment:
If there isn't a catalog it seems that sqlglot returns an empty string but we are expecting None